wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/ast2wdlom/EnhancedDraft3Ast.scala

```diff
@@ -26,10 +29,11 @@ object EnhancedDraft3Ast {
 
   implicit class EnhancedAst(val ast: Ast) extends AnyVal {
 
-    def getAttributeAsAstNodeVector(attr: String): Checked[Vector[AstNode]] = for {
-      attributeNode <- Option(ast.getAttribute(attr)).toChecked(s"No attribute $attr found on Ast of type ${ast.getName}")
-      asVector <- attributeNode.astListAsVector
-    } yield asVector
+    def getAttributeAsAstNodeVector(attr: String): Checked[Vector[AstNode]] = Option(ast.getAttribute(attr)) match {
+      case Some(a) => a.astListAsVector
+      case None if ast.getAttributes.containsKey(attr) => Vector.empty.validNelCheck
+      case None => s"No attribute $attr found on Ast of type ${ast.getName}".invalidNelCheck
```

Comment:
Did you still need this after we realized what we really wanted was `getAttributeAsOptionalAst `?